include/image-commands.mk

```diff
@@ -153,3 +153,10 @@ define Build/sysupgrade-nand
 		--rootfs $(word 2,$^) \
 		$@
 endef
+
+define Build/kernel2minor
+	if [ -x $(STAGING_DIR_HOST)/bin/kernel2minor ]; then \
+		$(STAGING_DIR_HOST)/bin/kernel2minor -k $@ -r $@.new $(1); \
+		mv $@.new $@; \
+	fi
```

Comment:
I think you can remove the check for the existence of kernel2minor here, let it fail hard if it is not present